python/core/auto_generated/qgspropertycollection.sip.in

```diff
@@ -86,7 +86,7 @@ Returns a matching property from the collection, if one exists.
 .. seealso:: :py:func:`hasProperty`
 %End
 
-    virtual QVariant value( int key, const QgsExpressionContext &context, const QVariant &defaultValue = QVariant() ) const = 0;
+    virtual QVariant value( int key, const QgsExpressionContext &context, const QVariant &defaultValue = QVariant(), bool *ok /Out/ = 0 ) const = 0;
```

Comment:
Isn't this an API break? Python code will get a tuple now where it used to get a single value

Reply:
yes.  I added SIP_PYARGREMOVE now.

Reply:
which doesn't work with overrides

Reply:
Add a v2 version?